### ISSUE TITLE
what if user input stupid high value 

### ISSUE THREAD

**Issue by frank0434** [frank0434/QuickTestMassBalanceCalculator#10]
![image](https://user-images.githubusercontent.com/30789416/68801700-73d4c480-06c1-11ea-855d-57264b58a674.png)

```
An error has occurred. Check your logs or contact the app author for clarification
```
**causes?**

```
Listening on http://127.0.0.1:36687
2019-11-13T20:28:37.877228+00:00 shinyapps[1209617]: Warning in `<reactive:Qtest.30.2>`(...) :
2019-11-13T20:28:37.877230+00:00 shinyapps[1209617]:   NAs introduced by coercion to integer range
2019-11-13T20:28:37.880094+00:00 shinyapps[1209617]: Warning: Error in if: missing value where TRUE/FALSE needed
136: <reactive:table_soil_N> [/srv/connect/apps/QTMB_dev/server.R#437]
2019-11-13T20:28:37.887501+00:00 shinyapps[1209617]:   169: mutate.data.frame
2019-11-13T20:28:37.887550+00:00 shinyapps[1209617]:   120: table_soil_N
2019-11-13T20:28:37.887501+00:00 shinyapps[1209617]:   167: function_list[[k]]
2019-11-13T20:28:37.887551+00:00 shinyapps[1209617]:   114: exprFunc [/srv/connect/apps/QTMB_dev/server.R#512]

2019-11-13T20:28:37.887548+00:00 shinyapps[1209617]:   159: <reactive:soil_filter> [/srv/connect/apps/QTMB_dev/server.R#220]
2019-11-13T20:28:37.887553+00:00 shinyapps[1209617]:    94: origRenderFunc
```

**Comment by frank0434** [comment]
integer in R ranges from -2147483647 to 2147483647 ~~about 2 billion~~ since the 32-bit data structures. 
values exceed the range will be coerced to NAs


https://www.r-bloggers.com/r-in-a-64-bit-world/

**Comment by frank0434** [comment]
fix by fe3ed84f055c948ad24635bb491f48498e357b0c
